governance-exercises/VoteWizard_poll_exercise_04/Emergency Response Poll - Risk mitigation options - August 9, 2023.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
In general I think you could do much better here. There are simpler ways of laying out this decision, and feels like you spent some time on writing out custom outcomes. 

That said, I don't think there is anything here that makes the poll invalid or misleading, I just think this would have taken you longer than was ideal.